db/migrate/20211126181518_create_branch.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBranch < ActiveRecord::Migration[6.1]
+  def change
+    create_table :branches do |t|
+      t.string :address
+      t.numeric :contact
```

Comment:
what is numeric?

Reply:
numeric will create a column of type decimal with precison: 10, so used this for contact number.

Reply:
changed to string